### ISSUE TITLE
[Docs] Confusion about the score range of the Simcc Head

### ISSUE THREAD

**Issue by T-S-Liang** [open-mmlab/mmpose#3207]
### 📚 The doc issue

Dear all,

Thank you for the great work! I’ve been confused about the range for the Simcc head for a long time. I’ve seen similar issues Duplicate of #2939 , but the details for applying specific scores for COCO evaluation still seem to be unclear.

The following are my doubts:

1. The score value for the Simcc head seems not to be normalized. In the evaluation code from mmpose, a keypoint threshold is needed for filtering out the low quality predictions, would there be any standard guideline to implement this?

2. I’m wondering if there would be a standard way for incorporating the keypoint score for evaluation. As far as I can understand from the several projects like RTMPose and DWPose, it seems that the the output score is considered as the maximum of the output logits (which is not normalized). Should we apply the logits maximum or the normalized value as the score for final evaluation?

Thank you so much!

**Comment by weichaoyu18** [comment]
I've run into this issue too. I found that the simcc head trains on a distribution, so the raw output logits have no constraints and range between (-inf, inf), which means they cannot be used directly as scores. If you use the result after softmax as the score, you will find that the score is very small, because the ground truth (gt) is generated using a Gaussian distribution rather than a one-hot distribution. Therefore, I think for evaluation, we can only use the result after softmax as the score, and then set an empirical threshold for evaluation.
